salt/modules/win_pkg.py

```diff
@@ -1346,32 +1448,27 @@ def remove(name=None, pkgs=None, version=None, **kwargs):
     '''
     Remove the passed package(s) from the system using winrepo
 
-    :param name:
-        The name of the package to be uninstalled.
-    :type name: str, list, or None
-
-    :param str version:
-        The version of the package to be uninstalled. If this option is used to
-        to uninstall multiple packages, then this version will be applied to all
-        targeted packages. Recommended using only when uninstalling a single
-        package. If this parameter is omitted, the latest version will be
-        uninstalled.
-
-    Multiple Package Options:
-
-    :param pkgs:
-        A list of packages to delete. Must be passed as a python list. The
-        ``name`` parameter will be ignored if this option is passed.
-    :type pkgs: list or None
-
     .. versionadded:: 0.16.0
 
-    *Keyword Arguments (kwargs)*
-    :param str saltenv: Salt environment. Default ``base``
-    :param bool refresh: Refresh package metadata. Default ``False``
+    Args:
+        name (str): The name(s) of the package(s) to be uninstalled. Can be a
+            single package or a comma delimited list of packages, no spaces.
```

Comment:
Multiline paramater docs need to start on their own line or they are not displayed properly. Needs to be:
```
Args:
    name (str):
        The name(s) of the packages(s)...
```